Maestro/subscriptions.json

```diff
@@ -1554,66 +1554,66 @@
         "https://github.com/aspnet/SignalR-Client-Cpp/blob/release/**/*",
         "https://github.com/aspnet/AspLabs/blob/release/**/*",
         "https://github.com/dotnet/aspnetcore/blob/release/2/**/*",
-        // "https://github.com/dotnet/aspnetcore/blob/release/3.0/**/*",
-        // "https://github.com/dotnet/aspnetcore/blob/release/3.1/**/*",
+        "https://github.com/dotnet/aspnetcore/blob/release/3.0/**/*",
+        "https://github.com/dotnet/aspnetcore/blob/release/3.1/**/*",
         "https://github.com/dotnet/aspnetcore-tooling/blob/release/2/**/*",
-        // "https://github.com/dotnet/aspnetcore-tooling/blob/release/3.0/**/*",
-        // "https://github.com/dotnet/aspnetcore-tooling/blob/release/3.1/**/*",
+        "https://github.com/dotnet/aspnetcore-tooling/blob/release/3.0/**/*",
+        "https://github.com/dotnet/aspnetcore-tooling/blob/release/3.1/**/*",
         "https://github.com/dotnet/blazor/blob/release/2/**/*",
-        // "https://github.com/dotnet/blazor/blob/release/3.0/**/*",
-        // "https://github.com/dotnet/blazor/blob/release/3.1/**/*",
+        "https://github.com/dotnet/blazor/blob/release/3.0/**/*",
```

Comment:
You can just remove all the 3.0 entries here.

Reply:
Done